en/includes/quick-starts/mcp-server.md

```diff
@@ -22,7 +29,15 @@ Note down the following values from the **Protocol** tab of the registered appli
 - **`client-secret`** from the **Protocol** tab. 
 - **The name of your {{ product_name }} organization**
 ---
-Next, go to the **API Authorization** tab in your Asgardeo application settings and click the **Authorize an API Resource** button. In the pop-up window, use the dropdown to search and select each required resource from the Asgardeo Management API by typing the first few letters of the resource name. For each selected resource, make sure you select **authorized scopes** to match those listed in the table below.
+Next, you need to authorize the API resources and scopes required for the actions performed by the MCP tools. Each tool
+may require different levels of access, so you can selectively authorize only the API resources and scopes relevant to
+your intended use. To do this, navigate to the **API Authorization** tab of your Asgardeo application and click **Authorize an
+API Resource**. In the pop-up that appears, use the dropdown to search and select the required API Resources from the
+Asgardeo Management API by typing the first few letters of each resource name. For every selected API Resource, ensure
+you assign the appropriate **authorized scopes**. For guidance on which API Resources and Scopes are needed by each tool,
+see the [API Resources and Scopes Required for MCP Tools]({{base_path}}/references/mcp-tool-api-resource-access/) section.
+
+The following table lists the required API resources and scopes for all currently supported tools:
 
```

Comment:
I think it's important to show mapping of permissions to tools here rather than hiding that.
What if we bring the table you have pointed in a different location to here and replace existing ?

Reply:
And we can add a note saying, you can control access for only for a set of tools by authorizing only the required set of permissions for the mcp server client